### ISSUE TITLE
Unable to get QR code from API

### ISSUE THREAD

**Issue by manojsitapara** [open-wa/wa-automate-nodejs#1866]
I am using wa-automate-docker, before today's new release I was able to get QR code on web browser from my server, but now I am unable to get QR code. 

NOTE: I can see QR code in terminal

Earlier I used to get QR code like below
http://localhost:8080/qr

After scanning QR code code direct from terminal I am able to access api-docs (http://localhost:8080/api-docs/), but problem is cannot get QR from browser.

@smashah can you please help me?

**Comment by smashah** [comment]
@manojsitapara it's probably because you're using an old image. Can you pull the latest image and try again please.

Nowadays every single release of the main library results in a new docker image

**Comment by manojsitapara** [comment]
Yes I did pull latest image

**Comment by cleytoncoro** [comment]
Maybe is related to last whastapp update. Try this:

Inside "/src/lib/wapi.js" u need to find this line:
```
else webpackChunkbuild.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);
```
and then replace for:
```
else webpackChunkwhatsapp_web_client.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);
```

I don't use qrcode here, then i can't test it.

**Comment by smashah** [comment]
> Maybe is related to last whastapp update. Try this:
> 
> Inside "/src/lib/wapi.js" u need to find this line:
> 
> ```
> else webpackChunkbuild.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);
> ```
> 
> and then replace for:
> 
> ```
> else webpackChunkwhatsapp_web_client.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);
> ```
> 
> I don't use qrcode here, then i can't test it.

This was fixed in v 4.8.2

Try 

```bash
> docker pull openwa/wa-automate
> docker run openwa/wa-automat
```

@manojsitapara can you show me a screenshot of your logs please

**Comment by manojsitapara** [comment]
Yes I did same, let me try again

![image](https://user-images.githubusercontent.com/12126671/125147858-bb884980-e14b-11eb-94f9-9050cbe38038.png)

`docker run -e PORT=8080 -p 8080:8080 openwa/wa-automate:latest`

on terminal I can see QR code 

![image](https://user-images.githubusercontent.com/12126671/125147842-94ca1300-e14b-11eb-8e0b-4741e6786545.png)

but when accessing QR from browser unable to get it

**Comment by smashah** [comment]
@manojsitapara 

Try with

```bash
> docker run -e PORT=8080 -p 8080:8080 openwa/wa-automate:latest --popup
```

**Comment by manojsitapara** [comment]
@smashah how stupid I am !! I am sorry, I forget --popup

**Comment by manojsitapara** [comment]
Do we need to update existing docker file with -- popup?
https://github.com/open-wa/wa-automate-docker/blob/master/Dockerfile

**Comment by smashah** [comment]
@manojsitapara if you as a user assume that popup is already set then maybe it's a good idea to update dockerfile to reflect the expectation. Please make an issue there and I will add it in the morning

**Comment by manojsitapara** [comment]
Okay sure. Thank you
